go.mod

```diff
@@ -3,24 +3,12 @@ module github.com/szuecs/routegroup-client
 go 1.15
```

Comment:
Should we bump this?

Reply:
That's the minimal required go version. Do we really care about that?
As long as the code compiles seems to be fine, no?